final_task/rss_reader/rss_reader.py

```diff
@@ -0,0 +1,45 @@
+import feedparser
+
+
+def get_dict_from_rss_site(source):
+    """Function for getting dict from RSS URL"""
+    try:
+        dict_from_rss_site = feedparser.parse(source)
+        if dict_from_rss_site:
+            return dict_from_rss_site
+        else: return ConnectionError(f"{source} can't be reached")
+    except OSError:
+        raise ConnectionError(f"{source} can't be reached")
+
+
+def check_key_in_dict_from_rss_site(dict_from_rss_site, key):
+    """Function for checking existing of key in dict"""
+    if dict_from_rss_site.get(key, None):
+        return True
+    return False
```

Comment:
It is better to use `in` operator instead of this function. For example:
`if key in dict_from_rss_site:`